artiq/coredevice/sampler.py

```diff
@@ -129,6 +129,32 @@ def sample_mu(self, data):
             data[i] = val >> 16
             val &= 0xffff
             data[i - 1] = -(val & mask) + (val & ~mask)
+            
+    
+    @kernel
+    def sample_mu_write(self, data):
+        """prepare ADC.
```

Comment:
Please write accurate and quality comments. They will appear in the documentation. "prepare ADC" isn't what this does at all.